Misc/NEWS.d/next/Core and Builtins/2021-07-06-22-22-15.bpo-44490.BJxPbZ.rst

```diff
@@ -0,0 +1,2 @@
+Fix ``typing`` module issues with ``types.Union`` parameters and arguments
+resolving. Patch provided by Yurii Karabas.
```

Comment:
Seems a little vague, here's a suggestion:
```suggestion
:mod:`typing` now searches for type parameters in ``types.Union`` objects.
``get_type_hints`` will also properly resolve annotations with nested
``types.Union`` objects. Patch provided by Yurii Karabas.
```
Also please rename your issue to something more specific. Maybe "Improve typing module compatibility with types.Union" ?